### PR TITLE
Update supported distros in `v0.71` Release Notes

### PR DESCRIPTION
**Pull request by ndokos**
PBENCH-811

Add list of supported/unsupported distros to release notes.

### CODE REVIEW

File: doc/RELEASE_NOTES.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ We did not bump the "major" release version number with these changes because we
 
 Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
 
+This release supports RHEL 7.9, RHEL8.6, RHEL 9, CentOS-Stream 8, CentOS-Stream 9 and Fedora 35. For various reasons, it does *NOT* support RHEL 8.x (where x <= 5), RHEL 9.1 or Fedora 36. If you need support for any of these, please talk to us: we will do our best to accommodate you in some way, but there is no guarantee.
```
Comment (dbutenhof) [inline doc/RELEASE_NOTES.md:33]: I have hesitated for minutes to even question this, but I'm uncomfortable with dumping these all into the same bucket:

- RHEL 8.4 / 8.5 are problems due to the ansible/ansible-core changes, which aren't well supported in the standard repos ... but you have a workaround by enabling additional support repos. I'm not sure I'd want the release notes to go into detail, but they could be called out as having "known workarounds"?
- RHEL 9.1 is just "immature" and doesn't have all the support repos available yet, a problem likely to be resolved by time though some adjustments on our part might be necessary?
- Fedora 36 and Python3.10 present enormous issues we haven't yet figured out how to deal with; pretty sure at this point we don't even want to suggest we're willing to get drawn into that, making me wary of the "we will do our best to accommodate you" line, which seems more appropriate for the RHEL 8.x/9.1 entries...

**Review by webbnh — DISMISSED** [review]

File: doc/RELEASE_NOTES.md (hunk 1 above)
Comment (ndokos) [reply doc/RELEASE_NOTES.md:33]: The weasel words are "For various reasons ..." :-) I wanted the release notes to contain the "what", not the "why" - but if there is some consensus, I'm open to changing the wording.

Comment (ndokos) [reply doc/RELEASE_NOTES.md]: I added short parenthetical explanations of the various reasons.

File: doc/RELEASE_NOTES.md [hunk 2]
```diff
@@ -38,7 +39,7 @@ There are no installation changes in this release: see the [Getting Started Guid
 
 After installation or update, you should have version `0.71.0-3g0b7f55850` of the `pbench-agent` RPM installed.
 
-RPMs are available from [Fedora COPR](https://copr.fedorainfracloud.org/coprs/ndokos/pbench/), covering Fedora 34, 35, & 36 (`x86_64` only), EPEL 7, 8, & 9 (`x86_64` and `aarch64`), and CentOS Stream 8 & 9 (`x86_64` and `aarch64`).
+RPMs are available from [Fedora COPR](https://copr.fedorainfracloud.org/coprs/ndokos/pbench/), covering Fedora 35, & 36 (`x86_64` only), EPEL 7, 8, & 9 (`x86_64` and `aarch64`), and CentOS Stream 8 & 9 (`x86_64` and `aarch64`), but there are problems with some distros (see above).
```
[the remark this answers is not in the document]
Comment (ndokos) [reply doc/RELEASE_NOTES.md]: Not sure that `x < 6` is better, but I took both of your suggestions above.

File: doc/RELEASE_NOTES.md [hunk 3]
```diff
@@ -30,6 +30,7 @@ We did not bump the "major" release version number with these changes because we
 
 Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent` release (tagged as `b0.69-bp`).  A number of bug fixes and behaviors from the `v0.71` work have already been back-ported and delivered in the various `v0.69.*` releases since then.  These release notes will highlight only the behavioral changes that have not been back-ported previously.
 
+This release supports RHEL 7.9, RHEL8.6, RHEL 9, CentOS-Stream 8, CentOS-Stream 9 and Fedora 35. For various reasons, it does *NOT* support RHEL 8.x for x < 6 (ansible vs. ansible-core dependency problem), RHEL 9.1 (missing repos) or Fedora 36 (python-3.10 problems). If you need support for any of these, please talk to us: we will do our best to accommodate you in some way, but there is no guarantee.
```
Comment (webbnh) [inline doc/RELEASE_NOTES.md:33]: Should we have a space in "RHEL 8.6", like we have in the previous and subsequent monikers?

**Review by webbnh — APPROVED** [review]
> I added short parenthetical explanations of the various reasons.

["She compromised my principles!"](https://www.youtube.com/watch?v=UrGw_cOgwa8&t=21s)

**Review by webbnh — APPROVED** [review]
👍

**Review by npalaska — DISMISSED** [review]

**Review by riya-17 — DISMISSED** [review]

File: doc/RELEASE_NOTES.md (hunk 1 above)
Comment (ndokos) [reply doc/RELEASE_NOTES.md]: @dbutenhof, can you check the change and see if that resolves the converstaion? I'd like to merge this.

Comment (ndokos) [reply doc/RELEASE_NOTES.md]: NM  - I just made an update to get the latest commits into the list.  This PR will be the first commit *after* the "official" tag point of the release in any case.

**Comment by ndokos** [comment]
Pushed another commit to include the latest commits into the changelog. This should be the final version of the release notes (modulo review comments).

**Review by webbnh — APPROVED** [review]
👍

**Review by npalaska — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]
I'm not sure offhand how I'd write it more clearly, but at first glance the "RHEL 8.x (blah blah blah)" is likely to read as "we don't support RHEL 8". I'm not going to add a conversation or request changes, but I have the vague feeling that some sort of clever word smithing might improve this. 🍤

**Comment by ndokos** [comment]
Added the "last" changes and reworded the 8.x explanation a bit.

**Review by webbnh — APPROVED** [review]
👍

File: doc/RELEASE_NOTES.md [hunk 4]
```diff
@@ -30,15 +30,16 @@ We did not bump the "major" release version number with these changes because we
...
 Installation
 ====
 
 There are no installation changes in this release: see the [Getting Started Guide](https://distributed-system-analysis.github.io/pbench/gh-pages/start.html) for how to install or update.
 
-After installation or update, you should have version `0.71.0-3g0b7f55850` of the `pbench-agent` RPM installed.
+After installation or update, you should have version `0.71.0-3g85910732a` of the `pbench-agent` RPM installed.
 
-RPMs are available from [Fedora COPR](https://copr.fedorainfracloud.org/coprs/ndokos/pbench/), covering Fedora 34, 35, & 36 (`x86_64` only), EPEL 7, 8, & 9 (`x86_64` and `aarch64`), and CentOS Stream 8 & 9 (`x86_64` and `aarch64`).
+RPMs are available from [Fedora COPR](https://copr.fedorainfracloud.org/coprs/ndokos/pbench/), covering Fedora 35, & 36 (`x86_64` only), EPEL 7, 8, & 9 (`x86_64` and `aarch64`), and CentOS Stream 8 & 9 (`x86_64` and `aarch64`), but please note there are problems with some distros as described above.
```
Comment (portante) [inline doc/RELEASE_NOTES.md:42]: Let's remove the reference to Fedora 36 here.

**Review by portante — CHANGES_REQUESTED** [review]

**Review by dbutenhof — APPROVED** [review]

File: doc/RELEASE_NOTES.md (hunk 4 above)
Comment (ndokos) [reply doc/RELEASE_NOTES.md]: Removed.

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**